main/view/src/main/webapp/WEB-INF/classes/coceso.properties.default

```diff
@@ -20,7 +20,7 @@ js.plugins={}
 
 # Geocode DB config
 geocode.db.driver=org.postgresql.Driver
-geocode.db.url=jdbc:postgresql://localhost:5432/geocode
+geocode.db.url=jdbc:postgresql://localhost:5432/coceso
```

Comment:
in the default setting the coceso backend and the geocode addon write to different databases. therefore the different example was mentioned here.

Reply:
Shall I undo this change (and fix the README.md to be clear about the layout of the databases)?

Reply:
yes please